Dockerfile

```diff
@@ -181,20 +181,13 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
 # Build VMware Tools
-ENV OVT_VERSION 9.10.0
+ENV OVT_VERSION 10.0.0-3000743
 
-# use fixes for post 3.19 and 4.0 kernels from https://github.com/davispuh/open-vm-tools/tree/fixed and http://git.io/vIXpn
-# rebased onto the 9.10.0 release
-ENV OVT_REPO       https://github.com/cloudnativeapps/open-vm-tools
-ENV OVT_BRANCH     stable-9.10.x-kernel4-vmhgfs-fix
-ENV OVT_COMMIT     f654bdb390dd6753985379ea7df058aa8f6294ee
+RUN curl --retry 10 -fsSL "https://github.com/vmware/open-vm-tools/archive/open-vm-tools-${OVT_VERSION}.tar.gz" | tar -xz --strip-components=1 -C /
```

Comment:
That `-C /` is suspicious looking too, but I see that it's because the repo has kind of a strange structure. :+1: